packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1801,6 +1801,19 @@ class MessageProperty extends DiagnosticsProperty<void> {
        super(name, null, description: message, style: style, level: level);
 }
 
+/// Debugging message for DevTools deep links.
```

Comment:
Move this class from diagnostics.dart to widget_inspector.dart. This isn't general purpose enough to be in foundation